SeaBlock/control.lua

```diff
@@ -202,6 +185,19 @@ script.on_load(
   end
 )
 
+script.on_event(defines.events.on_player_created,
+  function (e)
+    if game.is_multiplayer() then
```

Comment:
Add check if `starting_items` is nil (like how there is a check in `create_rock_chest`). 
[Brave New World](https://mods.factorio.com/mod/brave-new-world) mod sets this to nil via Sea Block's remote interface.
![image](https://user-images.githubusercontent.com/59639/176377994-1d3ee144-40d7-4af2-a6c7-a88f938c0917.png)

```suggestion
    if global.starting_items and game.is_multiplayer() then
```